client/src/components/Nav Menu/NavMenu.css

```diff
@@ -0,0 +1,91 @@
+.nav-container {
```

Comment:
Looks like you have a space in the folder name? Can you make it all one work (no spaces)?

Reply:
Done.